src/main/kotlin/org/jitsi/nlj/stats/PacketStreamStats.kt

```diff
@@ -78,5 +79,12 @@ class PacketStreamStats {
          * Total number of packets.
          */
         val packets: Long
-    )
+    ) {
+        fun toJson() = OrderedJsonObject().apply {
+            put("bitrate_bps", bitrate.bps)
+            put("packetrate", packetRate)
+            put("total_bytes", bytes)
+            put("total_packets", packetRate)
```

Comment:
Should be `packets.get()` here, and `packetRate.rate` above? And `bytes.get()`?

Reply:
No, because this method is on the `PacketStreamStats.Snapshot` class.  `get()` is already called when creating the snapshot.

Reply:
Should be `packets` is right, thanks.

Reply:
Yes, I got confused. Should be `packets` instead of `packetRate`, though.